subsys/net/l2/openthread/Kconfig

```diff
@@ -36,7 +37,6 @@ config OPENTHREAD_DTLS
 	select MBEDTLS_SSL_EXPORT_KEYS
 	select MBEDTLS_CTR_DRBG_ENABLED
 	select MBEDTLS_HMAC_DRBG_ENABLED
```

Comment:
I know that some of this is legacy but are we not able to use TinyCrypt instead of mbed for OpenThread?

Reply:
No, OpenThread does not abstract crypto functions, it has direct dependencies to mbedTLS.